onnxruntime/core/optimizer/constant_folding.cc

```diff
@@ -16,11 +16,11 @@ namespace onnxruntime {
 
 ConstantFolding::ConstantFolding(const IExecutionProvider& execution_provider,
                                  bool skip_dequantize_linear,
-                                 const std::unordered_set<std::string>& compatible_execution_providers,
+                                 const InlinedHashSet<std::string_view>& compatible_execution_providers,
                                  const std::unordered_set<std::string>& excluded_initializers) noexcept
```

Comment:
>const std::unordered_set<std::string>& excluded_initializers [](http://example.com/codeflow?start=33&length=60)

could this be changed to an InlinedHashSet too?

Reply:
This fans out changes a lot.